openstack/baremetalintrospection/v1/introspection/results.go

```diff
@@ -174,10 +175,11 @@ type Data struct {
 // Sub Types defined under Data and deeper in the structure
 
 type BaseInterfaceType struct {
-	ClientID string `json:"client_id"`
-	IP       string `json:"ip"`
-	MAC      string `json:"mac"`
-	PXE      bool   `json:"pxe"`
+	ClientID      string                 `json:"client_id"`
+	IP            string                 `json:"ip"`
+	MAC           string                 `json:"mac"`
+	PXE           bool                   `json:"pxe"`
+	LldpProcessed map[string]interface{} `json:"lldp_processed"`
```

Comment:
nit: `LLDPProcessed`?

Reply:
Yeah, if we do `ID`, we should do `LLDP`